examples/preact-kitchen-sink/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "preact-example",
+  "version": "4.1.0-alpha.10",
+  "private": true,
+  "scripts": {
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+    "build-storybook": "build-storybook -s public",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
+    "storybook": "start-storybook -p 9009 -s public"
+  },
+  "dependencies": {
+    "global": "^4.3.2",
+    "preact": "^8.3.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.1.2",
+    "@storybook/addon-actions": "4.1.0-alpha.10",
```

Comment:
please use only supported addons here

Reply:
The dependencies are updated to only display the supported addons.